### DISCUSSION TITLE
i cant "visit us again and reapply.

### DISCUSSION THREAD

**Discussion by thewLHE** [community/community#118184] [GitHub Education]
### Hi everyone,

i sent my info and university img and approved it. but i get email about i have a problem.. 
'Unfortunately, we were unable to verify your academic affiliation based on the information you submitted' like it
![email](https://github.com/community/community/assets/97024891/b0390e3a-af93-40cb-b41a-a242e03c429b)
so.. i check the issue and retry to get benefit to students. but continue button isnt working... what should i do?
![image](https://github.com/community/community/assets/97024891/ed2064a3-020e-41d6-b320-10fe89892a5e)

**Comment by samus-aran** [comment]
It appears that you have been approved. I suspect the email was sent in error. If you check your application status page the latest submission should say Approved 👍 

I'll be closing this discussion due to lack of recent activity. We can confirm that you have since been successful - enjoy your journey through GitHub Education.
